OrbitGl/TracepointTrack.cpp

```diff
@@ -81,4 +115,35 @@ float TracepointTrack::GetHeight() const {
 
 bool TracepointTrack::HasTracepoints() const {
   return !GOrbitApp->GetCaptureData().GetTracepointsOfThread(thread_id_).empty();
+}
+
+void TracepointTrack::OnPick(int x, int y) {
+  Vec2& mouse_pos = mouse_pos_[0];
+  canvas_->ScreenToWorld(x, y, mouse_pos[0], mouse_pos[1]);
+  mouse_pos_[1] = mouse_pos_[0];
+  picked_ = true;
+}
+
+void TracepointTrack::OnRelease() { picked_ = false; }
+
+std::string TracepointTrack::GetSampleTooltip(PickingId id) const {
+  static const std::string unknown_return_text = "Linux tracepoint event information missing";
```

Comment:
merely nitpicking:
Can be made ` constexpr std::string kUnknownReturnText`

Reply:
I will delete it completely since we should never be in a situation in which the information wouldn't be available.